<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Menu</class>
 <widget class="QMainWindow" name="Menu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>585</width>
    <height>555</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Menu</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Iconresfinal.qrc">
    <normaloff>:/iconres/icons/house menu.png</normaloff>:/iconres/icons/house menu.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="addser_bt">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>141</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Service</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cost_all_bt">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>320</y>
      <width>141</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>Cost of all services</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addben_bt">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>190</y>
      <width>141</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>Add benefits to residents</string>
    </property>
   </widget>
   <widget class="QPushButton" name="findser_onres_bt">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>190</y>
      <width>141</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>Find your services
consumed</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addres_bt">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>141</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Resident</string>
    </property>
   </widget>
   <widget class="QPushButton" name="alltable_bt">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>60</y>
      <width>141</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>Show database</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addser_on_res_bt">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>60</y>
      <width>141</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>Add services to 
existing residents</string>
    </property>
   </widget>
   <widget class="QPushButton" name="imex_bt">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>320</y>
      <width>141</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>Import/Export data</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>585</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="Iconresfinal.qrc"/>
 </resources>
 <connections/>
</ui>
